app/src/main/java/org/mozilla/fenix/search/awesomebar/AwesomeBarView.kt

```diff
@@ -82,14 +79,9 @@ interface AwesomeBarInteractor {
  */
 class AwesomeBarView(
     private val container: ViewGroup,
-    val interactor: AwesomeBarInteractor
-) : LayoutContainer {
-    val view: BrowserAwesomeBar = LayoutInflater.from(container.context)
```

Comment:
As discussed, we don't need this any more because we have this logic [here](https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt#L71-L75)